src/openzaak/management/commands/migrate_vestigingen_to_nnps.py

```diff
@@ -0,0 +1,80 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2025 Dimpact
+from django.core.management import BaseCommand
+from django.db import transaction
+
+from vng_api_common.constants import RolTypes
+
+from openzaak.components.zaken.models import NietNatuurlijkPersoon, Vestiging
+
+
+class Command(BaseCommand):
+    help = "Migrate deprecated Vestiging rollen to NietNatuurlijkePersoon."
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        self.stdout.write(
+            f"Attempting to migrate {Vestiging.objects.count()} Vestigingen to NietNatuurlijkePersonen..."
+        )
+
+        migration_count = 0
+        for vestiging in Vestiging.objects.iterator():
+
+            if NietNatuurlijkPersoon.objects.filter(rol=vestiging.rol).exists():
+                self.stdout.write(
+                    self.style.WARNING(
+                        f"Vestiging: {vestiging.handelsnaam[0]} could not be migrated, "
+                        f"a NietNatuurlijkPersoon already exists on Rol: {vestiging.rol.uuid}"
```

Comment:
Let's also log the pk so that the object is easier to find in the database if manual action is needed 
```suggestion
                        f"Vestiging (id={vestiging.id}): {vestiging.handelsnaam[0]} could not be migrated, "
                        f"a NietNatuurlijkPersoon already exists on Rol: {vestiging.rol.uuid}"
```